region/wreckedship/main/Bowling Alley.json

```diff
@@ -1318,6 +1319,52 @@
       ],
       "clearsObstacles": ["A"]
     },
+    {
+      "link": [6, 1],
+      "name": "Grapple Block, Crystal Flash, X-Ray Climb",
+      "requires": [
+        {"notable": "Grapple Block Crystal Flash"},
+        "canUseGrapple",
+        "canJumpIntoRespawningBlock",
+        "canTrickyJump",
+        {"or": [
+          "HiJump",
+          {"and": [
+            "canTrickySpringBallJump",
+            "canInsaneJump"
+          ]},
+          {"and": [
+            "canTrickySpringBallJump",
+            {"thornHits": 1}
+          ]},
+          {"and": [
+            "canSpringBallJumpMidAir",
+            {"not": "f_DefeatedPhantoon"},
+            "canRiskPermanentLossOfAccess"
+          ]}
+        ]},
+        "h_bombThings",
+        "h_CrystalFlash",
+        "canXRayClimb",
+        {"or": [
+          "h_fourTileJumpMorph",
+          "h_useSpringBall",
+          "canIBJ",
+          "SpeedBooster",
+          {"and": [
+            {"not": "f_DefeatedPhantoon"},
```

Comment:
Is this to climb farther left in the room?

Reply:
Yep. I wasnt sure if it was worth mentioning, since the midair morph isnt bad. There could also be strats that take spike hits while using the left crumble blocks, but it seems like overkill.